static/js/markdown_config.js

```diff
@@ -0,0 +1,54 @@
+/*
+    This config is in a separate file for partly
+    tactical reasons.  We want the webapp to
+    configure this one way, but we don't want to
+    share this code with mobile.
+
+    I also wanted to make some diffs clear before
+    doing any major file moves.
+
+    Also, I want the unit tests for markdown to
+    be able to reuse this code easily (and therefore
+    didn't just put this in ui_init.js).
+
+    Once the first steps of making markdown be a
+    shared library are complete, we may tweak
+    the file organization a bit.
+*/
+
+exports.get_helpers = () => {
+    return {
+        // HTML escaping
+        escape: _.escape,
+
+        // user stuff
+        get_user_by_name: people.get_by_name,
+        get_user_by_id: people.get_by_user_id,
+        is_known_user_id: people.is_known_user_id,
+        is_my_user_id: people.is_my_user_id,
```

Comment:
This can be replaced with `=== my_user_id()`.

Reply:
The function `is_my_user_id` has a long sordid history dating back to when we were fixing all the places the codebase was using `email` as a key.  More recently, it was defending against type confusion between strings and ints.  I think we're ready to kill this off completely, as it's had a `blueslip.error` clause for a couple weeks now, and I'm not aware of any errors being triggered.

I'm not quite ready to sweep that for the entire codebase (and I'm not implying you were suggesting that, to be clear, I'm just digressing), but I will definitely fix it in the markdown code.

Thanks!